spec/support/command_helpers.rb

```diff
@@ -373,6 +373,14 @@ def wait_normal_control
     )
   end
 
+  def wait_normal_control_and_status
+    wait_for_status(@task,"controlmode startup", [{'sCI'=>'S0020','n'=>'controlmode','s'=>'startup'}])
+    wait_for_status(@task,"signalmode eeeee", [{'sCI'=>'S0001','n'=>'signalgroupstatus','s'=>'eeeee'}])
+    wait_for_status(@task,"signalmode ffffffff", [{'sCI'=>'S0001','n'=>'signalgroupstatus','s'=>'ffffffff'}])
```

Comment:
what we expect is a sequence of modes like e,f,g. wait_for_status() will not complain if other modes are seen first. so a sequence like e,f,e,g will be accepted, which i don't think is correct. each group needs to follow the exact sequence of modes.
one idea is a helper like wait_for_status_change() where to pass the current mode, and the next mode as arguments. the status is then expected to stay in the current mode, until it changes to the next mode. if other modes is seen, it's an error